packages/@lwc/ssr-compiler/src/index.ts

```diff
@@ -8,7 +8,13 @@
 import compileJS from './compile-js';
 import compileTemplate from './compile-template';
 
-interface TransformOptions {}
+export interface JsTransformOptions {
+    name: string;
+    namespace: string;
+}
+export interface TemplateTransformOptions {
```

Comment:
Once/if `ssr-compiler` is the official path forward, we might want to define this interface as a subset of the compiler's [`TransformOptions`](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/compiler/src/options.ts#L82-L106), possibly by pulling it out of both into `@lwc/shared` or something like that.

For now, I think defining the interface here works well and should be left as is.